### DISCUSSION TITLE
How to use the new "tag" option

### DISCUSSION THREAD

**Discussion by njt1982** [JamesIves/github-pages-deploy-action#1191] [Q&A] [answered]
The README just mentions that "tag" is an option, but there are no examples of usage.

The [PR does have an example](https://github.com/JamesIves/github-pages-deploy-action/pull/1142) - but it seems to be hard coded to `v0.1`.

I'm probably missing something very obvious 😉 but it's not clear how you could use this without having to commit an update to your `deploy.yml` action every time you want to bump the tag...

Is there anyway to set it to an automatic value? Some kind of auto incrementing counter?

**Comment by JamesIves** [comment, marked as answer]
There's a few ways you can use them, one of which is using a workflow dispatch input: https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/ 

The intention is not to hardcode it, but have it be computed from either an input or a previous action run.

**Reply by njt1982** [reply]
Ahh - so there is no way to "auto tag" using this? You'd need some previous step to "calculate" the next tag?

On BitRise. I can set the app version to `v1.{build_number}` - is there a plugin to this kind of thing with Actions?

**Reply by JamesIves** [reply]
There probably is, but not I'm aware of sorry 😞 

If you think the tagging feature can be improved I'd be happy to discuss ways to make it better.

**Reply by JamesIves** [reply]
There probably is, but one not I'm aware of sorry 😞 

If you think the tagging feature can be improved I'd be happy to discuss ways to make it better.

**Reply by njt1982** [reply]
No need to apologise - your (FREE!) plugin is immensely useful and I am grateful for your efforts :) 

In terms of future changes / feature requests... I wonder if we can leverage the `GITHUB_RUN_NUMBER` env variable somehow? Maybe have a previous step that uses the run number to tag the "build" and then that gets passed via a variable (somehow?) to this tag parameter like you do in the link above with input variables... 🤷🏻‍♂️
